flow/connectors/mongo/metered_dialer.go

```diff
@@ -0,0 +1,45 @@
+package connmongo
```

Comment:
could we consolidate MeteredDialer with mysql's metered_dialer? it looks like there's a lot of overlap between the two